src/containers/buttons/AddTestButton.js

```diff
@@ -0,0 +1,55 @@
+import React, {Component} from 'react';
+import {connect} from "react-redux";
+import {bindActionCreators} from 'redux';
+import {addTestToProject, clearTests, hideUrlInput, clearUrlsList, hideTestNameInput, initTestName} from "../../actions"
+
+class AddTest extends Component {
+
+  addTestToProject() {
+    if(this.props.test.length > 0 && this.props.urls.length > 0) {
+      document.querySelectorAll(".block").forEach((block) => {
+        block.classList.add("block-hidden");
+      });
+      this.props.initTestName();
+    }
+  }
+  
+  checkIfTestAlreadyExists(testName) {
+    let exists = false;
+    document.querySelectorAll(".tests-container>.test-item>.test-name").forEach((test) => {
+      if(testName.name === test.innerHTML) {
+        exists = true;
+      }
+    });
+    return exists;
+  }
+
+  render () {
+    if(this.props.test.length > 0 && this.props.urls.length > 0) {
+      return (
+        <div className="add-test test-button" onClick={() => this.addTestToProject()}>ADD TEST</div>
```

Comment:
Wouldn't here just adding btn-disabled conditionally suffice to accomplish that?

Reply:
Done. Looks much cleaner, thanks!